scripts/report/01-make_reports.py

```diff
@@ -450,6 +451,13 @@ def run_report_average(session):
     evokeds = mne.read_evokeds(evoked_fname)
     fs_subjects_dir = config.get_fs_subjects_dir()
 
+    df_events = count_events(config.bids_root)
+    rep.add_htmls_to_section(
+        f'<center>{df_events.to_html()}</center>',
```

Comment:
Let's make it easier to style this via CSS later
```suggestion
        f'<div class="event-counts">\n    {df_events.to_html()}\n</div>',
```